docs/src/main/asciidoc/_cloud/Getting_Started_on_OpenShift.adoc.adoc

```diff
@@ -0,0 +1,145 @@
+# Getting Started with WildFly on OpenShift
```

Comment:
@jmfinelli name of the file is not correct (`.adoc.adoc`)

Reply:
ops...thanks @jmesnil